front/app/components/Avatar/index.tsx

```diff
@@ -142,7 +143,12 @@ const Avatar = memo(
 
     if (isLinkToProfile && hasValidProfileLink) {
       return (
-        <Link to={profileLink}>
+        <Link
```

Comment:
It is not a good idea to have a path and onClick on a link component at the same time, it feels very confusing. What if, instead, we add a `scrollToTop` prop to the link component that handles this functionality in the component itself.